subprojects/plugins/src/integTest/groovy/org/gradle/java/compile/AbstractGroovyCompileAvoidanceIntegrationSpec.groovy

```diff
@@ -24,49 +24,49 @@ abstract class AbstractGroovyCompileAvoidanceIntegrationSpec extends AbstractJav
 
     private String goodAstTransformation() {
         """
-import org.codehaus.groovy.transform.*;
-import org.codehaus.groovy.ast.*;
-import org.codehaus.groovy.control.*;
-@GroovyASTTransformation(phase = CompilePhase.CANONICALIZATION)
-public class MyASTTransformation extends AbstractASTTransformation {
-    @Override
-    public void visit(ASTNode[] nodes, SourceUnit source) {
-            System.out.println("Hello from AST transformation!");
-    }
-}
-    """
+            import org.codehaus.groovy.transform.*;
+            import org.codehaus.groovy.ast.*;
+            import org.codehaus.groovy.control.*;
+            @GroovyASTTransformation(phase = CompilePhase.CANONICALIZATION)
+            public class MyASTTransformation extends AbstractASTTransformation {
+                @Override
+                public void visit(ASTNode[] nodes, SourceUnit source) {
+                    System.out.println("Hello from AST transformation!");
+                }
+            }
+        """
     }
 
     private String goodAstTransformationWithABIChange() {
         """
-import org.codehaus.groovy.transform.*;
-import org.codehaus.groovy.ast.*;
-import org.codehaus.groovy.control.*;
-@GroovyASTTransformation(phase = CompilePhase.CANONICALIZATION)
-public class MyASTTransformation extends AbstractASTTransformation {
-    @Override
-    public void visit(ASTNode[] nodes, SourceUnit source) {
-            System.out.println("Hello from AST transformation!");
-    }
-    public void foo() { }
-}
-    """
+            import org.codehaus.groovy.transform.*;
+            import org.codehaus.groovy.ast.*;
+            import org.codehaus.groovy.control.*;
+            @GroovyASTTransformation(phase = CompilePhase.CANONICALIZATION)
+            public class MyASTTransformation extends AbstractASTTransformation {
+                @Override
+                public void visit(ASTNode[] nodes, SourceUnit source) {
+                    System.out.println("Hello from AST transformation!");
+                }
+                public void foo() { }
+            }
+        """
     }
 
     private String badAstTransformationNonABIChange() {
         """
-import org.codehaus.groovy.transform.*;
-import org.codehaus.groovy.ast.*;
-import org.codehaus.groovy.control.*;
-@GroovyASTTransformation(phase = CompilePhase.CANONICALIZATION)
-public class MyASTTransformation extends AbstractASTTransformation {
-    @Override
-    public void visit(ASTNode[] nodes, SourceUnit source) {
-        assert false: "Bad AST transformation!"
-    }
-    public void foo() { }
-}
-    """
+        import org.codehaus.groovy.transform.*;
```

Comment:
In order to be consistent with the other ones, this needs to be indented once more, right?

Reply:
Indeed, thanks